src/test/regress/expected/multi_subquery_complex_reference_clause.out

```diff
@@ -1823,6 +1961,104 @@ LIMIT 5;
        6
 (1 row)
 
+-- outer part of the LEFT JOIN consists only reference tables, so we cannot push down
```

Comment:
Well, once we support these queries, the author would probably curse at me for adding these queries. Still, I feel more comfortable with having a variety of failure cases. Let me know if you think otherwise